cairo_programs/cairo_0/bitwise_builtin.cairo

```diff
@@ -0,0 +1,16 @@
+%builtins bitwise
+
+from starkware.cairo.common.bitwise import bitwise_and, bitwise_xor, bitwise_or, bitwise_operations
```

Comment:
remove this

Reply:
Done! ✅

Reply:
Ty, @zmalatrax  I already fixed that, built it, and ran the tests, I think we are good now!
Also, I updated my branch with the main